src/wayback/tests/test_client.py

```diff
@@ -755,13 +772,19 @@ def test_timeout_applied_request(self, mock_class):
     @mock.patch('requests.Session.send', side_effect=return_timeout)
     def test_timeout_empty(self, mock_class):
         # Disabling default timeout
-        session = WaybackSession(timeout=None)
+        session = WaybackRequestsAdapter(timeout=None)
         res = session.request('GET', 'http://test.com')
         assert res.text == 'None'
         # Overwriting the default
         res = session.request('GET', 'http://test.com', timeout=1)
         assert res.text == '1'
 
+    @mock.patch('requests.Session.send', side_effect=return_user_agent)
+    def test_user_agent(self, mock_class):
+        adapter = WaybackRequestsAdapter()
+        agent = adapter.request('GET', 'http://test.com').text
+        assert agent.startswith('wayback/')
```

Comment:
I feel lucky I even noticed we needed this! We probably should have had a test before — the previous implementation was subtle because it used built-in functionality of `requests.Session` that changed when we stopped subclassing that.